src/uwtools/tests/drivers/test_driver.py

```diff
@@ -423,6 +443,45 @@ def test_Driver_runscript(arg, driverobj, type_):
         assert isinstance(runscript.call_args.kwargs[arg], type_)
 
 
+def test_driver_show_output(capsys, config):
+    ConcreteDriverTimeInvariant(config).show_output()
+    expected = """
+    {
+      "bar": [
+        "/path/to/bar1",
+        "/path/to/bar2"
+      ],
+      "foo": "/path/to/foo"
+    }
+    """
+    assert capsys.readouterr().out.strip() == dedent(expected).strip()
+
+
+@mark.parametrize(
+    "base_file,update_values,expected",
+    [
+        (False, False, {}),
+        (False, True, {"a": 33}),
+        (True, False, {"a": 11, "b": 22}),
+        (True, True, {"a": 33, "b": 22}),
+    ],
+)
+def test_Driver__create_user_updated_config_base_file(
+    base_file, driverobj, expected, tmp_path, update_values
+):
+    path = tmp_path / "updated.yaml"
+    if not base_file:
+        del driverobj._config["base_file"]
+    if not update_values:
+        del driverobj._config["update_values"]
+    ConcreteDriverTimeInvariant._create_user_updated_config(
+        config_class=YAMLConfig, config_values=driverobj.config, path=path
+    )
+    with open(path, "r", encoding="utf-8") as f:
+        updated = yaml.safe_load(f)
+    assert updated == expected
```

Comment:
This was just incorrectly positioned in the module -- moved up from a bit lower.